sql/src/main/java/io/crate/lucene/match/OptionParser.java

```diff
@@ -161,6 +161,9 @@ private static Fuzziness fuzziness(@Nullable Object fuzziness) {
         if (fuzziness == null) {
             return null;
         }
+        if (fuzziness instanceof BytesRef) {
```

Comment:
I think it's safe to remove the instanceof check and always use BytesRefs.toString(..)